util/unit_test/test_00_ensure_llm-d-infra.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python3
```

Comment:
Test cases have been provided; I see there are some others as well. Shouldn't we be calling these on PR submission?

Reply:
Unit tests are, admittedly, messy and not even consistently called atm. I wholeheartedly agree with @kalantar that we should be calling those on CI/CD, but does not need to be done in this PR.